src/codeql.ts

```diff
@@ -75,26 +74,9 @@ async function runQuery(
     REMOTE_QUERY_PACK_NAME,
   ]);
 
-  let cur = `${databaseName}/results`;
-  let entries: fs.Dirent[];
-  while (
-    (entries = fs.readdirSync(cur, { withFileTypes: true })) &&
-    entries.length === 1 &&
-    entries[0].isDirectory()
-  ) {
-    cur = path.join(cur, entries[0].name);
-  }
-
-  if (entries.length !== 1) {
-    throw new Error(`Expected a single file in ${cur}, found: ${entries}`);
-  }
-
-  const entry = entries[0];
-  if (!entry.isFile() || !entry.name.endsWith(".bqrs")) {
-    throw new Error(`Unexpected file in ${cur}: ${entry.name}`);
-  }
-
-  fs.renameSync(path.join(cur, entry.name), bqrsFilePath);
+  const bqrsFilePath = path.join("results", "results.bqrs");
+  const tempBqrsFilePath = getBqrsFile(databaseName);
+  fs.renameSync(tempBqrsFilePath, bqrsFilePath);
```

Comment:
I dithered on posting a comment here about moving the rename inside the `getBqrsFile` method, but eventually decided it doesn't make much difference. So I wanted to note this as a point but I'm not requesting any changes.

The crucial point for me is that the `getBqrsFile` method isn't generating the BQRS file but it's just finding an existing file that CodeQL created earlier. So I think things make sense as they are and keeping the `rename` outside the method is good.

Reply:
Yeah, FWIW I went through a similar thought process myself 😂 I think either way would be okay 🤷🏽